app/helpers/reports/regions_helper.rb

```diff
@@ -1,7 +1,7 @@
 module Reports::RegionsHelper
   def sum_registration_counts(repository, *keys)
-    slug, user_id = keys
-    repository.monthly_registrations_by_user.dig(slug)
+    slug, user_id, diagnosis = keys
```

Comment:
I didn't quite follow the reason to accept arguments as a splatted array here (vs just regular keyword params). `diagnosis` can be defaulted to much more nicely if we could use arguments.

Reply:
We will make it keyword params, it would be both easier to read and use.